opm/utility/ECLGraph.cpp

```diff
@@ -274,6 +279,19 @@ namespace {
                 std::vector<T>
                 scatterToGlobal(const std::vector<T>& x) const;
 
+                /// Map input vector to active grid cells.
+                ///
+                /// \param[in] x Input vector, defined on the explicitly
+                ///              active cells, all global cells or some
+                ///              other subset (e.g., all non-neighbouring
+                ///              connections).
+                ///
+                /// \return Input vector mapped to active cells or unchanged
```

Comment:
This is unclear: I assume you mean that input vector is mapped to active cells iff it has the same size as the number of global cells, otherwise it is simply copied?